### PR TITLE
W-13081276 sharepoint 3.5

### PR DESCRIPTION
**Pull request by sathya0**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by frantisekglova — DISMISSED** [review]

File: sharepoint/3.5/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -47,6 +47,29 @@ image::sharepoint-online-config.png[Online Configuration]
...
+* Listener Config: The configuration for the HTTP listener that listens for requests on the access token callback endpoint.
+* Callback Path: The path of the access token callback endpoint.
+* Authorize Path: The path of the local HTTP endpoint that triggers the OAuth dance.
+
+== OAuth Client Credentials
+
+Configuring the connector to use OAuth Client Credentials authentication requires the following information:
+
+* Site Url: The URL for the target SharePoint instance.
+* Client ID: The client ID to call the API.
```
Comment (dhirsch1) [inline sharepoint/3.5/modules/ROOT/pages/index.adoc]: Suggest: The client ID used for to call the API
or
The client ID to use to call the API

File: sharepoint/3.5/modules/ROOT/pages/sharepoint-connector-reference.adoc [hunk 2]
```diff
@@ -64,6 +65,37 @@ include::connectors::partial$oauth2-description.adoc[]
...
+===== OAuth Client Credentials
+
+Uses client credentials for authentication.
+
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Disable SSL certificate validation a| Boolean |  Disables the CN (Common Name) check so that the server won't throw an exception if the HTTPS certificate is not signed by a trusted partner. To prevent this, you can disable the CN (Common Name) check. Avoid disabling this for production environments. |  `false` |
+| Site Url a| String |  URL for the target SharePoint instance. |  | x
+| Client ID a| String |  Client ID to call the API. |  | x
```
Comment (dhirsch1) [inline sharepoint/3.5/modules/ROOT/pages/sharepoint-connector-reference.adoc]: Same comment as earlier

File: sharepoint/3.5/modules/ROOT/pages/sharepoint-connector-reference.adoc [hunk 3]
```diff
@@ -64,6 +65,37 @@ include::connectors::partial$oauth2-description.adoc[]
...
+| Token URL a| String |  URL used to get an access token. |  | x
+| Scopes a| String |  Defined scopes for the client ID.|  | 
+| Keystore alias a| String |  Alias used to get the correct certificate from the keystore. |  | x
+| Path to keystore a| String | Path to the client's keystore with the private key and the certificate. |  | x
```
Comment (dhirsch1) [inline sharepoint/3.5/modules/ROOT/pages/sharepoint-connector-reference.adoc]: and the certificate > and certificate. (global)

File: sharepoint/3.5/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -47,6 +47,29 @@ image::sharepoint-online-config.png[Online Configuration]
 * Password: Enter the corresponding password.
 * Site URL: Enter the corresponding site URL.
 
+== OAuth v2.0 Authorization Code
+
+Configuring the connector to use OAuth v2.0 Authorization Code authentication requires the following information:
+
```
Comment (dhirsch1) [inline sharepoint/3.5/modules/ROOT/pages/index.adoc:53]: Please remove the periods

File: sharepoint/3.5/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -47,6 +47,29 @@ image::sharepoint-online-config.png[Online Configuration]
...
+* Site Url: The URL for the target SharePoint instance.
```
Comment (dhirsch1) [inline sharepoint/3.5/modules/ROOT/pages/index.adoc]: Remove periods for these list items - global

In the interest of brevity, I would also remove " The" from the start of the descriptions (not a must-do but something to be aware of in the future).

**Review by dhirsch1 — DISMISSED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**